yapf/styles/pep8.ini

```diff
@@ -0,0 +1,71 @@
+[style]
+
+# Align closing bracket with visual indentation.
+align_closing_bracket_with_visual_indent = true
+
+# The column limit.
```

Comment:
These comments are now duplicated with the help in style.py (so they could go out of sync, not really a problem, but I think they could be removed).